daml-lf/spec/daml-lf-1.rst

```diff
@@ -2564,8 +2728,11 @@ List functions
   predicate give as first argument.
 
 
-TextMap functions
-~~~~~~~~~~~~~~~~~
+Text map functions
+~~~~~~~~~~~~~~~~~~
+
+**Entries ordering**: All the operation above preserved the order of
+ the map according the lexicographic order of its keys.
```

Comment:
I'm not sure what it means to both "preserve" the order and also use lexicographic ordering. (I'm probably missing something.) If it's ordered lexicographically, then it's sorted, and it's not necessary to "preserve" anything; there's an invariant in place.